src/actions/metadata.js

```diff
@@ -31,6 +31,14 @@ export function updateFieldKey(namePrefix, fieldKey, newKey) {
   };
 }
 
+export function changeFieldValue(nameAttr, tempValue) {
+  return {
+    type: ActionTypes.CHANGE_FIELD_VALUE,
+    nameAttr,
+    tempValue
+  };
+}
+
```

Comment:
I prefer instantly saving without trying to click to outside of the input box. This is not user-friendly at all. Users will think they saved properly when they hit `ctrl+s` without knowing the save button is not activated at all because they didn't lose focus. Hotkeys should be time-saver. Let's not block it.

I started to think that we should better move all of the things that we do on blur (trimming, auto-casting) to the put actions. That seems safer and unavoidable. I can create a PR for that if you agree.